plugin.video.dailymotion_com/default.py

```diff
@@ -113,16 +126,18 @@ def favouriteUsers():
           match = re.compile('###USER###=(.+?)###THUMB###=(.*?)###END###', re.DOTALL).findall(content)
           for user, thumb in match:
             addUserFavDir(user, 'owner:'+user, 'sortVideos1', thumb)
+    xbmcplugin.setContent(pluginhandle, "addons")
+    xbmc.executebuiltin('Container.SetViewMode(500)')
```

Comment:
This isn't allowed.

Reply:
Corrected it to use only if user setup the relevant modes in settings